addon/plugins/article-structure-plugin/commands/insert-structure.ts

```diff
@@ -108,8 +109,28 @@ function findInsertionRange(args: {
     return false;
   };
   const nextContainerRange =
-    findNodes(doc, selection.from, true, false, filterFunction).next().value ??
-    findNodes(doc, selection.from, true, true, filterFunction).next().value;
+    findNodes({
```

Comment:
can we make this configurable from the host app, and default to searching everything (aka the old behavior)?